doc/source/building/working_with_images/pxe_live_iso_boot.rst

```diff
@@ -0,0 +1,115 @@
+.. _pxe_live_boot:
+
+Boot Live ISO Image from Network
+================================
+
+.. sidebar:: Abstract
+
+   This page provides further information for handling
+   ISO images built with KIWI and references the following
+   articles:
+
+   * :ref:`hybrid_iso`
+
+In KIWI live ISO images can be configured to boot via PXE.
+This requires a functional network boot server setup on the
+system which presents the live ISO image to the network.
+Details how to setup such a server can be found in
+:ref:`pxe-boot-server`
+
+After the live ISO was built as shown in :ref:`hybrid_iso`,
+the following configuration steps are required to boot from
+the network:
+
+1. Extract initrd/kernel From Live ISO
+
+   The PXE boot process loads the configured kernel and initrd from
+   the PXE server. Because of that those two files must be extracted
+   from the live ISO image and copied to the PXE server as follows:
+   
+   .. code:: bash
+
+       $ mount LimeJeOS-Leap-42.3.x86_64-1.42.3.iso /mnt
```

Comment:
I would not do that because we are intentionally referring this article to another one which builds this image with that name. See the abstract information. Not sure if you remember but we designed all chapters as a stack.

At some point in time we need to renew the image names. I'll implement a way to make this easy but it's out of scope for this pull request

Reply:
Ok, no problem. But then you should keep the version consistent with line 57 with the `vbladed` command.